index.html

```diff
@@ -55,7 +55,12 @@
     <h2 >
         The home of wallpepers
     </h2>
-    g
+    <div class="category">
+      <button id="abstract-btn" type="submit">Abstract</button>
```

Comment:
use the 4 images in the assets folder as a background image for the buttons 

or use ` <img>`  instead of `<button>`  and the `onClick="anyFunction()"` attribute with it